tests/weight_proof/test_weight_proof.py

```diff
@@ -485,58 +470,53 @@ async def test_weight_proof_extend_multiple_ses(self, default_1000_blocks, block
-        curr: BlockRecord = blocks[peak]
-        while True:
-            sub_height_to_hash[curr.height] = curr.header_hash
-            if curr.sub_epoch_summary_included is not None:
-                sub_epoch_summaries[curr.height] = curr.sub_epoch_summary_included
-            if curr.height == 0:
-                break
-            curr = blocks[curr.prev_hash]
-        assert len(sub_height_to_hash) == peak_height + 1
-        block_cache = BlockCache(blocks, headers, sub_height_to_hash, sub_epoch_summaries)
-        wpf = WeightProofHandler(DEFAULT_CONSTANTS, block_cache)
-        wp = await wpf._create_proof_of_weight(sub_height_to_hash[peak_height - 50])
-        valid, fork_point = wpf.validate_weight_proof_single_proc(wp)
-
-        await connection.close()
-        assert valid
-        print(f"size of proof is {get_size(wp)}")
-
-
-@pytest.mark.parametrize(
-    "height,expected",
-    [
-        (0, 3),
-        (5496000, 2),
-        (10542000, 1),
-        (15592000, 0),
-        (20643000, 0),
-    ],
-)
-def test_calculate_prefix_bits_clamp_zero(height: uint32, expected: int):
+    # TODO: Revisit this skipped test because `blocks` is now a Dict[bytes32, BlockRecord]
+
+    # @pytest.mark.skip("used for debugging")
+    # @pytest.mark.asyncio
+    # async def test_weight_proof_from_database(self) -> None:
+    #     connection = await aiosqlite.connect("path to db")
+    #     db_wrapper = DBWrapper2(connection, db_version=2)
+    #     block_store = await BlockStore.create(db_wrapper)
+    #     blocks = await block_store.get_block_records_in_range(0, 0xFFFFFFFF)
+    #     peak = len(blocks) - 1
+    #     peak_height = blocks[peak].height
+    #     headers = await block_store.get_header_blocks_in_range(0, peak_height)
+    #     sub_height_to_hash = {}
+    #     sub_epoch_summaries = {}
+    #     # peak_header = await block_store.get_full_blocks_at([peak_height])
+    #     if len(blocks) == 0:
+    #         return None, None
+
+    #     assert peak is not None
+
+    #     # Sets the other state variables (peak_height and height_to_hash)
+    #     curr: BlockRecord = blocks[peak]
+    #     while True:
+    #         sub_height_to_hash[curr.height] = curr.header_hash
+    #         if curr.sub_epoch_summary_included is not None:
+    #             sub_epoch_summaries[curr.height] = curr.sub_epoch_summary_included
+    #         if curr.height == 0:
+    #             break
+    #         curr = blocks[curr.prev_hash]
+    #     assert len(sub_height_to_hash) == peak_height + 1
+    #     block_cache = BlockCache(blocks, headers, sub_height_to_hash, sub_epoch_summaries)
+    #     wpf = WeightProofHandler(DEFAULT_CONSTANTS, block_cache)
+    #     wp = await wpf._create_proof_of_weight(sub_height_to_hash[peak_height - 50])
+    #     valid, _ = wpf.validate_weight_proof_single_proc(wp)
+
+    #     await connection.close()
+    #     assert valid
+    #     print(f"size of proof is {get_size(wp)}")
```

Comment:
instead of commenting this out, could it be moved into its own file under `tools`?
There are some other debugging utilities in there too